Initial build for RWAHS site with config options

mostro20 opened the pull request:
Have included both the zipped archive and the unzipped features areas. If you'd prefer just the zipped archive, let me know and I'll take out the other part. I think this should have all my config in it - never used features in git so excuse my mistakes if it's not quite what you are looking for and we can discuss on Monday.

mostro20 committed:
Initial build for RWAHS site with config options

Have included both the zipped archive and the unzipped features areas.

leftclickben commented:
I think we need the individual files but not the zip. Zip files are binary so they can't be reviewed. But it's redundant to have both. It might be worth looking at github releases to generate the zip. But in any case we shouldn't have the zip file itself under version control. @kehh let me know if there's a reason for an exception here.

leftclickben commented:
My other comment is about the namespace `initial_website_build`. Maybe this is my lack of understanding of drupal and features but this name seems to be something that will need to change in the future? What I mean is, as it evolves, it won't be so much of an "initial build". But I'm not sure about this one so feel free to correct me here.

kehh commented on initial_website_build/initial_website_build.info.yml:
```diff
@@ -0,0 +1,29 @@
+  - file
+  - filter
+  - image
+  - jssor
```
Nice slider - can't quite believe what you can do with it through builder, but I gather that's not enabled in the Drupal module.

kehh commented on initial_website_build/initial_website_build.info.yml:
```diff
@@ -0,0 +1,29 @@
+name: 'Initial website build'
```
I'd probably change this to `Website Configuration` and rename the package to `website_configuration`

kehh commented on initial_website_build/initial_website_build.info.yml:
```diff
@@ -0,0 +1,29 @@
+name: 'Initial website build'
+description: 'This is the first build of the RWAHS site, it is not refined yet and still requires considerable cleaning up of theming and content, but all the basic config has been completed.'
```
I'd replace this with `This feature contains the base configuration of the RWAHS site.`

The reset of this would be part of the description of the pull request, but the feature itself is something that evlolves over time.

kehh commented:
OK @mostro20 basically it's pretty much all ok (I really like the `yml` serialization of the features - much less verbose than the old fashioned php that 7.x generated).

The only change is to rename the feature and a better description.

mostro20 committed:
Website Configuration

Initial features build, with more refinements to come after client
feedback is received.

mostro20 commented:
Hey Kehan, this is my first update in an existing request, so hopefully my git noobness has been avoided and this is all cool.

kehh commented on website_configuration/config/install/core.entity_form_display.node.activity.default.yml:
```diff
@@ -0,0 +1,101 @@
+status: true
+dependencies:
+  config:
+    - field.field.node.activity.field_accession_date
```
Not sure what accession_date is on an activity.

kehh commented:
After our discussion yesterday I feel that this config is ok as a WIP. There will doubtless be further changes to the features before this can actually be installed on the site.
